plugins/interface/phy_intf_plugin.js

```diff
@@ -40,6 +40,15 @@ class PhyInterfacePlugin extends InterfaceBasePlugin {
     await exec("sudo systemctl daemon-reload");
   }
 
+  async prepareEnvironment() {
+    await super.prepareEnvironment();
+    if (this.networkConfig.enabled) {
+      const txRingBuffer = this.networkConfig.txBuffer || 4096;
+      const rxRingBuffer = this.networkConfig.rxBuffer || 4096;
+      await exec(`sudo ethtool -G ${this.name} tx ${txRingBuffer} rx ${rxRingBuffer}`).catch((err) => {});
+    }
+  }
+
```

Comment:
This should be a platform/gold related configuration.   these queues sizes are physical hardware queues, they different between different ethernet chips.

Reply:
You can set it in config